authorization/src/test.rs

```diff
@@ -5,18 +5,18 @@ use ed25519_dalek::Keypair;
 use rand::thread_rng;
 use soroban_sdk::testutils::ed25519::Sign;
 
-use soroban_sdk::{BytesN, Env, RawVal, Vec};
+use soroban_sdk::{BytesN, Env, RawVal, Symbol, Vec};
 use soroban_sdk_auth::{Ed25519Signature, SignaturePayload, SignaturePayloadV0};
 
-pub fn to_ed25519(e: &Env, kp: &Keypair) -> Identifier {
-    Identifier::Ed25519(kp.public.to_bytes().into_val(e))
-}
-
 fn generate_keypair() -> Keypair {
     Keypair::generate(&mut thread_rng())
 }
 
-fn make_auth(e: &Env, kp: &Keypair, args: Vec<RawVal>, function: &str) -> Signature {
+fn make_identifier(e: &Env, kp: &Keypair) -> Identifier {
+    Identifier::Ed25519(kp.public.to_bytes().into_val(e))
+}
```

Comment:
`to_ed25519` is a little ambiguous because it returns an Identifier, not an Ed25519 keypair, so the name suggests it does the opposite of what it actually does. Renamed `to_ed25519` to `make_identifier` so there's some naming consistency between it and `make_auth`.

Reply:
This naming is my fault, it originates from the very first tests I was writing back in June. We should probably fix this everywhere, it's pretty ubiquity. Might I recommend `make_ed25519_identifier` since there are other ways a keypair could be turned into an identifier (eg. as an account)?

Reply:
Should we add the ed25519 to teh other things too? 
```
generate_ed25519_keypair
make_ed25519_identifier
make_ed25519_signature
```

Reply:
I tried it out, the code is more noisy. I'm inclined to leave out the ed25519 everywhere in this example, because this example only does ed25519 auth, so it is implied.

Reply:
`generate_keypair` doesn't need to change since it returns a `KeyPair`. `make_auth` is only in this example and I think it's fine to keep it as is. 

`to_ed25519` is used in almost every example though, so we should keep that consistent. All of our examples use ed25519 Identifiers because it's simple, but we will add non-ed25519 tests at some point. I think `make_ed25519_identifier ` is fine, but if you want to shorten it a bit, we could do `to_ed25519_identifier` instead?